certified-connectors/Scancloud/apiDefinition.swagger.json

```diff
@@ -347,8 +347,8 @@
             "type": "string",
             "default": "",
```

Comment:
@Ajaatsecplugs empty properties are not allowed. You can remove it until required or define a value. Please refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-exact-operation-responses).